packages/core/src/Forms/Adornment/Adornment.tsx

```diff
@@ -28,6 +28,8 @@ export interface HvAdornmentProps
   onClick?: React.MouseEventHandler<HTMLButtonElement>;
   /** If this property is defined the adornment visibility will be exclusively controlled by this value. */
   isVisible?: boolean;
+  /** Whether the adornment is disabled or not. */
+  disabled?: boolean;
```

Comment:
I think we should be getting this value from the context `HvFormElementContext` below, same as `status`